.github/workflows/ci.yml

```diff
@@ -89,7 +110,7 @@ jobs:
   clippy:
     runs-on: ubuntu-latest
     env:
-      # Make sure CI fails on all warnings, including Clippy lints.
+      # Make sure CI fails on all warnings - including Clippy lints.
```

Comment:
this was just to trigger workflow with generated cach, don't mind this

Reply:
You can always re-run manually BTW, and it should have the cache